conf.py

```diff
@@ -276,7 +276,7 @@
 
 # The unique identifier of the text. This can be a ISBN number
 # or the project homepage.
-epub_identifier = 'https://github.com/hpcng/singularity-userdocs'
+epub_identifier = 'https://github.com/{Command}/{Command}-userdocs'
```

Comment:
No, URLs should use a different new replacement `{orgrepo}` as called for in #30

Reply:
Oh, that was a silly mistake. Fixing it.